tests/test_stl.py

```diff
@@ -162,6 +162,13 @@ def test_exp_optional():
     assert holder.member_initialized()
 
 
+@pytest.mark.skipif(not hasattr(m, "has_filesystem"), reason="no <filesystem>")
+def test_fs_path():
+    from pathlib import Path
+
+    assert m.parent_path(Path("foo/bar")) == Path("foo")
```

Comment:
There should probably be a test with a custom `PathLike` class, here?

Reply:
done

Reply:
Great!